CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 ## master
 
+- sidecar resources for multipart requests can be labeled with allowed or required
```

Comment:
why do we need both `allowed` and `required`.

It should be enough to signal the `required` parts